kpi/009.adoc

```diff
@@ -258,3 +256,4 @@ If the codelist implementation is not clear there should be additional free text
 `gmd:otherConstraints` or `gmd:useLimitation`.
 
 Further guidance on data policy implementation can be found in the link:https://library.wmo.int/doc_num.php?explnum_id=10257[Guide to WMO Information System (WMO No. 1061, Section 5.8.1.10)].
+
```

Comment:
Line 253-255: "In addtion to programmatic checks which will provide an score, it should be clear to the user what are the conditions of use for the resource published."

suggested revision

"In addition to programmatic checks which will provide a score, the conditions of use for the resource published should be clear to the user."